Dockerfile

```diff
@@ -251,6 +246,26 @@ RUN (PIP_ARGS=$([[ "$LOCALSTACK_PRE_RELEASE" == "1" ]] && echo "--pre" || true);
       pip3 install --upgrade ${PIP_ARGS} localstack-ext[runtime])
 RUN make entrypoints
 
+# Install packages which should be shipped by default
+RUN source .venv/bin/activate && \
+    python -m localstack.cli.lpm install --parallel 4 \
+      cloudformation-libs \
+      dynamodb-local \
+      iot-rule-engine \
+      kinesis-mock \
+      lambda-java-libs \
+      local-kms \
+      mqtt \
+      postgres \
+      redis \
+      stepfunctions \
+      stepfunctions-local \
+      timescaledb && \
+    rm -rf /tmp/localstack/* && \
+    rm -rf /var/lib/localstack/cache/* && \
+    chmod -R 777 /var/lib/localstack
```

Comment:
Not too sure if this approach is not too aggressive in general, but it's ok for me :)